doc/storage.md

```diff
@@ -10,11 +10,14 @@ discourse: 7735
 About storage </explanation/storage>
 Create a storage pool <howto/storage_create_pool>
 Create an instance in a pool <howto/storage_create_instance>
-Add a storage volume <howto/storage_add_volume>
-Configure storage settings <howto/storage_configure>
-List pools and volumes <howto/storage_list>
-Move or copy volumes <howto/storage_move>
-Back up storage volumes <howto/storage_backup>
-Resize storage <howto/storage_resize>
+Create a storage volume <howto/storage_create_volume>
+Configure storage pool <howto/storage_configure_pool>
+Configure storage volume <howto/storage_configure_volume>
+List pools and show configuration <howto/storage_list_pools>
+List volumes and show configuration <howto/storage_list_volumes>
```

Comment:
This doesn't fit into one line in the navigation - I think it's sufficient to have just "List storage pools/volumes" here. The full title can mention the show configuration as well.
Or we could have "Show storage pools/volumes" instead.